wolfssl/internal.h

```diff
@@ -4365,6 +4366,7 @@ struct WOLFSSL {
 #ifdef WOLFSSL_EARLY_DATA
     EarlyDataState earlyData;
     word32 earlyDataSz;
+    word32 earlyDataStatus;
```

Comment:
Change type to byte.

Reply:
Thank you for pointing it out. Changed it to byte.